junction/proposals/forms.py

```diff
@@ -166,3 +172,23 @@ def __init__(self, conference, *args, **kwargs):
         self.fields['proposal_section'].choices = _get_proposal_section_choices(conference)
         self.fields['proposal_type'].choices = _get_proposal_type_choices(conference)
         self.fields['reviewer_comment'].choices = ProposalReviewerComment.CHOICES
+
+
+class ProposalVotesFilterForm(forms.Form):
```

Comment:
`ProposalVotesFilterForm` and `ProposalsToReviewForm` have some common fields. Can we reuse it ?